lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -487,15 +585,18 @@ func loadSubscribers(networkID string, includeSub subscriberFilter, keys ...stri
 	return subs, nil
 }
 
-func loadAllSubscribers(networkID string) (map[string]*subscribermodels.Subscriber, error) {
-	mutableSubs, err := loadAllMutableSubscribers(networkID)
+func loadSubscriberPage(networkID string, pageSize uint32, pageToken string) (map[string]*subscribermodels.Subscriber, string, error) {
+	mutableSubs, nextPageToken, err := loadMutableSubscriberPage(networkID, pageSize, pageToken)
 	if err != nil {
-		return nil, err
+		return nil, nextPageToken, err
```

Comment:
`return nil, "", err` -- when there's an err returned from a fn, the caller assumes all other values are garbage. So let's enforce that expectation generally speaking